### DISCUSSION TITLE
SLOGAN

### DISCUSSION THREAD

**Discussion by gabrielsousarod** [EasyRoomCompany/Guardian#91] [Ideas]
1-Sua chave para reuniões eficientes e sem complicações. 

2-Sua sala, rápida, prática e na palma da sua mão.

3-Agende com facilidade, reuniões de sucesso!

O segundo tá muito bom. o que acham?

**Comment by mikaelmonteirodev** [comment]
Fala @gabrielsousarod! No caso ela seria em inglês, certo? Sendo assim seria legal ver como ficaria. 👍

**Reply by gabrielsousarod** [reply]
1-Your key to efficient, hassle-free meetings.

2-Your room, fast, practical and in the palm of your hand.

3-Easily schedule successful meetings!

**Reply by gabrielsousarod** [reply]
lembrando que ele sempre leva a tradução pra quarto :(

**Comment by kevinDsousa** [comment]
Gostei da segunda opção. @ivannaprata @fabianopmelo o que acham?

**Comment by mikaelmonteirodev** [comment]
My friends, como não custa nada perguntar. E se uma quarta opção fosse:

> Easy, extremely easy...

Será que não arrancaria uns sorrisos? 😄

**Reply by ivannaprata** [reply]
Gostei também. Então a opção 2 e essa.

**Comment by ivannaprata** [comment]
Gosto da segunda opção.
